torchrl/data/replay_buffers/rb_prototype.py

```diff
@@ -14,6 +14,23 @@
 
 
 class ReplayBuffer:
+    """
+    #TODO: Description of the ReplayBuffer class needed.
```

Comment:
Is the `TODO` part needed?

Reply:
I can take it or leave it. Do you mean to replace it with just `Description of the ReplayBuffer class needed.` ?